operator/v1/types_ingress.go

```diff
@@ -716,8 +720,61 @@ type AWSNetworkLoadBalancerParameters struct {
 	// +optional
 	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
 	Subnets *AWSSubnets `json:"subnets,omitempty"`
+
+	// eipAllocations assign Elastic IP addresses to the Network Load Balancer.
```

Comment:
This field doesn't really mention outright that this contains a list of IDs. I think should at least mention that it's IDs somewhere.

```suggestion
	// eipAllocations is a list of IDs for Elastic IP (EIP) addresses that
        // are assigned to the Network Load Balancer.
```

Use your judgment on what you think is best.

Reply:
yes that sounds fine